mitodl/bootcamp-ecommerce
# Replacing Github Actions with Travis

#### What are the relevant tickets?
https://github.com/orgs/mitodl/projects/2

#### What's this PR do?
Uses Github Actions instead of Travis

#### How should this be manually tested?
Anyone can simply push a branch to see if tests are running in GitHub actions and test coverage is being uploaded or not.

**Side Note**: This PR also includes some work regarding **Remove tox, use pytest instead** as done in the following PR https://github.com/mitodl/bootcamp-ecommerce/pull/1072
